### PR TITLE
feat: link to `Add Wallet` if no wallets created

### PR DESCRIPTION
**Pull request by pete-watters**
This PR makes the Action Bar dynamic so that if no wallets have been created it shows `Add Wallet` and links to `APP_ROUTES.WalletCreateNewWallet`

### CODE REVIEW

**Review by kyranjamie — CHANGES_REQUESTED** [review]
Why does this return a anon function, rather than a component that accepts props? 🤔

Underneath the JSX, a component is just a function that accepts arguments, as props, so afaik it isn't necessary to wrap in a fn like this.

File: apps/mobile/src/app/wallet/(tabs)/_layout.tsx [hunk 1]
```diff
@@ -187,7 +218,7 @@ export default function TabLayout() {
           }}
         />
       </Tabs>
-      {createFilledActionBar(ref, router)()}
+      {CreateFilledActionBar(ref, router)()}
```
Comment (kyranjamie) [inline apps/mobile/src/app/wallet/(tabs)/_layout.tsx:221]: I see this pattern was here from before, but very odd imo and a react anti-pattern, unless there's some RN quirk I'm not aware of

Comment (kyranjamie) [reply apps/mobile/src/app/wallet/(tabs)/_layout.tsx:221]: image

**Comment by pete-watters** [comment]
> Why does this return a anon function, rather than a component that accepts props? 🤔
> 
> Underneath the JSX, a component is just a function that accepts arguments, as props, so afaik it isn't necessary to wrap in a fn like this.

I went for getting it working quickly using what was there already but I can refactor it 👍

**Review by kyranjamie — APPROVED** [review]
Way cleaner, thanks @pete-watters

**Comment by pete-watters** [comment]
> Way cleaner, thanks @pete-watters

No problem. I'll make more effort to refactor as I go

**Comment by pete-watters** [comment]
> Looks awesome @pete-watters !! We will also need this action bar on home screen. Also, can you, please, attach a video of UI changes? It's a bit easier to review that way as a reviewer shouldn't change branches and build app on that PR that way.

Ok, I will make sure to add a video in future and add it to the Home Screen

**Pull request merged**